spinalcordtoolbox/utils/shell.py

```diff
@@ -72,10 +72,8 @@ def display_viewer_syntax(files, verbose, im_types=[], minmax=[], opacities=[],
     display_viewer_syntax([file1, file2, file3])
     display_viewer_syntax([file1, file2], im_types=['anat', 'softseg'], minmax=['', '0,1'], opacities=['', '0.7'])
     """
-    # If the file's absolute path is relative to the CWD, then use just the relative path to that file.
-    files = [str(abspath.relative_to(Path.cwd())) if abspath.is_relative_to(Path.cwd())
-             # Otherwise, the file must be outside the working directory, so use the absolute path.
-             else str(abspath) for abspath in [Path(f).absolute() for f in files]]
+    # Try to convert the path to one that is relative to the CWD; if not possible, use the abspath instead.
+    files = [str(relpath_or_abspath(filepath, parent_path=os.getcwd())) for filepath in files]
```

Comment:
I see, switching from `Path.cwd()` to `os.getcwd()` lets us get rid on the `pathlib` import altogether. Nice!